netebpfext/net_ebpf_ext.c

```diff
@@ -757,37 +771,73 @@ net_ebpf_extension_uninitialize_wfp_components(void)
 {
     size_t index;
     NTSTATUS status;
+    int max_retries = 10;
+    LARGE_INTEGER timeout = {0};
+    timeout.QuadPart = -100000000; // 10 seconds.
```

Comment:
Windows has macros such as `XMSTO100NS(_millis)`  to make it easier to read and write these values, and also make the units explicit. Eight zeros does seem correct for 10 seconds in 100-ns.